tests/integration/test_case.go

```diff
@@ -129,6 +129,51 @@ type SchemaUpdate struct {
 	// The schema update.
 	Schema string
 
+	// PolicyIDs is an optional list argument which makes it easier to substitute/embed PolicyIDs
+	// into the schema string where place holder labels ("%policyID%") are at, without us using
+	// explicit policy identifiers.
+	//
+	// Note:
+	// - The number of placeholder labels (%policyID%) must exactly match the number of policyID
+	//   indexes in this list.
+	// - The list must contain the policyID Indexes in the same order they are to be substituted in.
+	// - Can substitute the same policyID at an index, by specifying that index multiple times.
+	// - The indexes must be valid (correspond to the number of policies added so far).
+	//
+	// Example:
+	//
+	// Consider we have one policy that was added resulting in the following policyID:
+	// PolicyID="94eb195c0e459aa79e02a1986c7e731c5015721c18a373f2b2a0ed140a04b454"
+	//
+	// Then using this attribute like:
+	// PolicyIDs: immutable.Some([]int{0, 0}),
+	//
+	// On a Schema string like:
+	// ```
+	//	type Users1 @policy(id: "%policyID%", resource: "users") {
+	//		name: String
+	//		age: Int
+	//	}
+	//
+	//	type Users2 @policy(id: "%policyID%", resource: "users") {
+	//		name: String
+	//		age: Int
+	//	}
+	// ```
+	// The Schema that will be loaded will be this modified one:
+	// ```
+	//	type Users1 @policy(id: "94eb195c0e459aa79e02a1986c7e731c5015721c18a373f2b2a0ed140a04b454", resource: "users") {
+	//		name: String
+	//		age: Int
+	//	}
+	//
+	//	type Users2 @policy(id: "94eb195c0e459aa79e02a1986c7e731c5015721c18a373f2b2a0ed140a04b454", resource: "users") {
+	//		name: String
+	//		age: Int
+	//	}
+	// ```
+	PolicyIDs immutable.Option[[]int]
```

Comment:
suggestion: I don't think this being `immutable.Option` adds anything besides complexity - there appears to be no behavioural differences between `immutable.None` and an empty slice.

Same goes for Fred's suggestion about decoupling this from `policyID`.